io.c

```diff
@@ -13331,6 +13331,163 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
+ *
+ *  === Writing
+ *
+ *  - ::binwrite:: Writes the given string to the file at the given filepath,
+                   in binary mode.
+ *  - ::write:: Writes the given string to +self+.
+ *  - ::<<:: Appends the given string to +self+.
+ *  - #print:: Prints one or more objects to +self+.
+ *  - #printf:: Formats and prints one or more objects to +self+.
+ *  - #putc:: Writes a character to +self+.
+ *  - #puts:: Writes lines to +self+.
+ *  - #pwrite:: Writes the given string at the given offset,
+ *              not updating the offset.
+ *  - #write:: Writes one or more given strings to +self+.
+ *  - #write_nonblock:: Writes one or more given strings to +self+ in non-blocking mode.
+ *
+ *  === Copying
+ *
+ *  - ::copy_stream:: Copies some or all data from src (filepath or \IO-like object)
+ *                    to dst (filepath or \IO-like object).
+ *
+ *  === Positioning
+ *
+ *  - #lineno:: Returns the current line number in +self+.
+ *  - #lineno=:: Sets the line number is +self+.
+ *  - #pos:: Returns the current offset (bytes) in +self+.
+ *  - #pos=:: Sets the offset (bytes) in +self+.
+ *  - #reopen:: Reassociates +self+ with a new or existing I/O stream.
+ *  - #rewind:: Positions +self+ to the beginning of input.
+ *  - #seek:: Sets the offset for +self+.
+ *  - #tell:: Returns the current offset (bytes).
+ *
+ *  === Iterating
+ *
+ *  - ::foreach:: Calls the given block with each successive file line
+ *                for the given filepath.
+ *  - #each (aliased as #each_line):: Calls the given block
+ *                                    with each successive line in +self+.
+ *  - #each_byte:: Calls the given block with each successive byte in +self+.
+ *  - #each_char:: Calls the given block with each successive character in +self+.
+ *  - #each_codepoint:: Calls the given block with each successive codepoint in +self+.
+ *  - #each_line:: Calls the given block with each successive codepoint in +self+.
+ *
+ *  === Converting
+ *
+ *  - ::try_convert:: Returns the new \IO object resulting from converting
+ *                    the given object.
+ *  - #inspect:: Returns the string representation of +self+.
+ *
+ *  === Settings
+ *
+ *  - #autoclose=:: Sets whether +self+ auto-closes.
+ *  - #binmode:: Sets +self+ to binary mode.
+ *  - #close_on_exec=:: Sets the close-on-exec flag.
+ *  - #sync=:: Sets the sync-mode to the given value.
+ *
+ *  === Querying
+ *
+ *  - #autoclose?:: Returns whether +self+ auto-closes.
+ *  - #binmode?:: Returns whether +self+ is in binary mode.
+ *  - #close_on_exec?:: Returns the close-on-exec flag for +self+.
+ *  - #closed?:: Returns whether +self+ is closed.
+ *  - #eof? (aliased as #eof):: Returns whether +self+ is at end-of-file.
+ *  - #fileno (aliased as #to_i):: Returns the integer file descriptor for +self+
+ *  - #sync:: Returns whether +self+ is in sync-mode.
+ *  - #tty (aliased as #isatty):: Returns whether +self+ is a terminal.
+ *
+ *  === Closing
```

Comment:
I would move all of these methods to Settings.

Reply:
> I would move all of these methods to Settings.

You mean section "Querying"?

Reply:
Oh, I see, you mean "Closing" to Settings, right?

Reply:
No.  The Closing section has methods that change the settings of the IO object, so they should go to Settings.  Querying is for operations that do not modify the file, correct?

Reply:
Now I think you mean "Querying" to "Settings."  Just like in _Idiocracy_, call me "Not Sure."  Will wait for you to clarify.

Reply:
Sorry about that.  I realize because of GitHub's display that this was easy is misinterpret.  You would think if I only comment on a single line, that's the only line GitHub would display, but I guess they assume you will always want the content above (but not below).

Reply:
Fixed.